### ISSUE TITLE
Flask Migrate Error

### ISSUE THREAD

**Issue by MOJarrett** [app-generator/flask-adminlte#5]
I followed the README steps precisely and installed the packages inside my virtual environment. When I try to run flask I get the following error:
```
   from   flask_migrate import Migrate
ModuleNotFoundError: No module named 'flask_migrate'
```

Here are the pip packages I am using within the virtual environment:
```
Package          Version
---------------- -------
alembic          1.7.7
aniso8601        9.0.1
attrs            21.4.0
click            8.1.3
dnspython        2.2.1
email-validator  1.1.3
Flask            2.0.2
Flask-Login      0.5.0
Flask-Migrate    3.1.0
Flask-Minify     0.37
Flask-MySQLdb    1.0.1
flask-restx      0.5.1
Flask-SQLAlchemy 2.5.1
Flask-WTF        1.0.0
greenlet         1.1.2
gunicorn         20.1.0
htmlmin          0.1.12
idna             3.3
itsdangerous     2.1.2
Jinja2           3.0.2
jsmin            3.0.1
jsonschema       4.5.1
lesscpy          0.15.0
Mako             1.2.0
MarkupSafe       2.1.1
mysqlclient      2.1.0
pip              22.0.4
ply              3.11
pyrsistent       0.18.1
python-dotenv    0.19.2
pytz             2022.1
rcssmin          1.1.0
setuptools       62.1.0
six              1.16.0
SQLAlchemy       1.4.29
Werkzeug         2.0.3
wheel            0.37.1
WTForms          3.0.0
xxhash           3.0.0
```

**Comment by app-generator** [comment]
Hello @MOJarrett, 

Can you tell me more about your environment? 

- OS
- Python version
- Terminal: `CMD`, or Powershell (for windows)
- The commands list

Ty for using our products.

**Comment by MOJarrett** [comment]
Thanks for responding so quickly

- OS - Mac OS Monterey
- Python version - 3.9.7
- Terminal: Mac OS

**The commands list:**

1. cd Documents
2. git clone https://github.com/app-generator/flask-adminlte.git
3. cd flask-adminlte
4. virtualenv env
5. source env/bin/activate
6. pip3 install -r requirements.txt
7. export FLASK_APP=run.py
8. export FLASK_ENV=development
9. flask run

Results in this error at http://127.0.0.1:5000/:
<img width="975" alt="image" src="https://user-images.githubusercontent.com/97242033/171229470-72c63612-add2-4dc9-9619-d3c5b76636fa.png">

Looking again at the README I can see the other section on creating a MYSQL DB. I haven't created it and updated the .env file with details of the DB instance. Could this be causing the issue I am seeing?

**Comment by app-generator** [comment]
Hello @MOJarrett 

As a temporary workaround, please disable the usage of `Flask_migrate`.

Just comment the lines `7`, and `29`:

```python
import os
#from   flask_migrate import Migrate   <--- HERE
from   flask_minify  import Minify
from   sys import exit

...

app = create_app(app_config)
#Migrate(app, db)                         <--- HERE

if not DEBUG:
    Minify(app=app, html=True, js=False, cssless=False) 
```

The app is not tested under `Anaconda` and this might be a problem because you have an upper environment layer that manages the dependencies. 

My advice is to use the app outside of your code editor. Just use the console and type all the commands as per README. 
In the meantime, we will investigate this issue. 

Let us know your progress.

**Comment by MOJarrett** [comment]
I worked out the issue. I tried the commenting out approach above and it lead to more lines showing as errors. That showed that flask was not running from the virtual environment correctly and for some reason was defaulting to use the local python environment.

I then uninstalled flask from my local MacBook python environment using `pip3 uninstall flask`. Now when running `flask run` from the virtual environment I got a different error which said I was missing the MySQL DB. Once a MySQL DB was running using the user, password and db name from the .env file everything worked fine.
